server/test/unit/server/webservices/views/test_util.py

```diff
@@ -0,0 +1,57 @@
+import json
+import mock
+import unittest
+
+from django.http import HttpResponse, HttpResponseNotFound
+
+from pulp.server.webservices.controllers.base import json_encoder as pulp_json_encoder
+from pulp.server.webservices.views import util
+
+
+class TestWebservicesUtils(unittest.TestCase):
+    """
+    Test webservices utilities.
+    """
+
+    def test_generate_json_response_default_params(self):
+        """
+        Make sure that the response is correct under normal conditions.
+        """
+        test_content = {'foo': 'bar'}
+        response = util.generate_json_response(HttpResponse, test_content)
+        self.assertTrue(isinstance(response, HttpResponse))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response._headers.get('content-type'),
+                         ('Content-Type', 'application/json'))
+        response_content = json.loads(response.content)
+        self.assertEqual(response_content, test_content)
+
+    def test_generate_json_response_not_found(self):
+        """
+        Test that response is correct for non-base HttpResponses
+        """
+        response = util.generate_json_response(HttpResponseNotFound)
+        self.assertTrue(isinstance(response, HttpResponse))
+        self.assertEqual(response.status_code, 404)
+        self.assertEqual(response._headers.get('content-type'),
+                         ('Content-Type', 'application/json'))
+
+    def test_generate_json_response_invalid_response_class(self):
+        """
+        generate_django_response should raise a TypeError for any responses that are not a
```

Comment:
Only the first line is displayed when tests are run with -v. If a test has a comment then the first line only is shown. I usually have a short single line sentence, and then use detail like this as another paragraph below in the docblock.